src/weather_model_graphs/create/archetype.py

```diff
@@ -119,23 +146,27 @@ def create_oskarsson_hierarchical_graph(
 
     Parameters
     ----------
-    xy_grid: np.ndarray
-        2D array of grid point positions.
-    grid_refinement_factor: float
-        Refinement factor between grid points and bottom level of mesh hierarchy
+    coords: np.ndarray
+        2D array of grid point positions, either in-projection Cartesian coordinates or lat-lons
+    mesh_node_distance: float
+        Distance (in x- and y-direction) between created mesh nodes in bottom level,
+        in coordinate system of coords
     level_refinement_factor: float
         Refinement factor between grid points and bottom level of mesh hierarchy
+    projection: cartopy.crs.CRS or None
```

Comment:
And there, I think this docstring line needs updating to reflect that `coords_crs` and `graph_crs` are now provided separately.